### ISSUE TITLE
Publish to pypi on merge to master

### ISSUE THREAD

**Issue by vsudilov** [vsudilov/flask-consulate#10]
This should be automated; we can set up travis to do this for us.

**Comment by sobolevn** [comment]
That's sounds nice. 

That's how I use this approach: every tag in the `master` branch should trigger a release.
Example: https://github.com/sobolevn/django-split-settings/blob/master/.travis.yml

**Comment by sobolevn** [comment]
Could you please also invite @lk-geimfari to collaborate?

**Comment by lk-geimfari** [comment]
If we want to publish package automatically, then we should create a new release before merge to master, else will raised publishing error:

```
error: Upload failed (400): A file named "package-0.0.1.tar.gz" already exists 
for package-0.0.1.tar.gz. 
```

Or (to publish package each time you add a new tag) use something like:

```
deploy:
  provider: pypi
  user: username
  password:
    secure: my_secure_password
  on:
    tags: true
    branch: master
```
Related stuff [here](http://5minutes.youkidea.com/howto-deploy-python-package-on-pypi-with-github-and-travis.html), [here](https://www.appneta.com/blog/pypi-deployment-with-travis-ci/) and [here](https://blog.travis-ci.com/2013-01-14-new-client/).

**Comment by vsudilov** [comment]
Closed via #18
